module/VuFindSearch/tests/unit-tests/src/VuFindTest/Backend/WorldCat/ConnectorTest.php

```diff
@@ -85,6 +85,8 @@ public function testGetHoldingsHttpFailure()
         $response = $this->createMock(\Laminas\Http\Response::class);
         $response->expects($this->any())->method('isSuccess')
             ->will($this->returnValue(false));
+        $response->expects($this->once())->method('getStatusCode')
```

Comment:
I'm a bit curious what this change has to do with PHP 8.1... or why it's necessary to change a test when the corresponding class didn't change.